lib/vc.js

```diff
@@ -162,7 +310,85 @@ function _getId(obj) {
   return obj.id;
 }
 
-async function _checkCredential({credential}) {
+/**
+ * @param presentation
+ * @throws {Error}
+ * @private
+ */
+function _checkPresentation(presentation) {
+  if(!presentation['@context']) {
```

Comment:
The cred/pres checks that `@context` and `type` have multiple values would also catch them not existing.  Perhaps the simple existence checks should be removed?  The multiple value error is also more informative on what is needed.

Reply:
Good point! Removed.